core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="skill-selector">
+  <div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+    <div class="list-group oppia-list-group">
+      <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
```

Comment:
layout="row" property was available in `md-card`. But no such property exist on `mat-card`

Reply:
Done, removed